src/mlpack/tests/fastmks_test.cpp

```diff
@@ -205,28 +205,29 @@ BOOST_AUTO_TEST_CASE(SparsePolynomialFastMKSTest)
     for (size_t j = 0; j < sparseIndices.n_rows; ++j)
     {
       if (std::abs(sparseKernels(j, i)) > 1e-15)
-        BOOST_REQUIRE_CLOSE(sparseKernels(j, i), denseKernels(j, i), 1e-5);
+        REQUIRE(sparseKernels(j, i) ==
+            Approx(denseKernels(j, i)).epsilon(1e-7));
```

Comment:
Do you mind to use `{}` here to improve readability?

Reply:
Done.